test/integration/StreamrClient.test.js

```diff
@@ -99,10 +99,14 @@ describe('StreamrClient', () => {
                     })
                 })
                 client.connection.on('UnicastMessage', (msg) => {
-                    streamMessage = msg.payload
-                    assert.strictEqual(streamMessage.parsedContent.test, 'client.publish with Stream object as arg')
+                    /* eslint-disable prefer-destructuring */
```

Comment:
Would be nicer to do as eslint wants here

Reply:
Doing object destructuring here throws a parsing error.